components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -93,6 +44,66 @@ const BillingSubscriptions: React.FC<Props> = () => {
     const [showWarning, setShowWarning] = useState(true);
     const [showInfo, setShowInfo] = useState(true);
 
+    const upgradeMattermostCloud = () => (
+        <div className='UpgradeMattermostCloud'>
+            <div className='UpgradeMattermostCloud__image'>
+                <img src={upgradeMattermostCloudImage}/>
+            </div>
+            <div className='UpgradeMattermostCloud__title'>
+                <FormattedMessage
+                    id='admin.billing.subscription.upgradeMattermostCloud.title'
+                    defaultMessage='Need more users?'
+                />
+            </div>
+            <div className='UpgradeMattermostCloud__description'>
+                <FormattedMarkdownMessage
+                    id='admin.billing.subscription.upgradeMattermostCloud.description'
+                    defaultMessage='The free tier is **limited to 10 users.** Get access to more users, teams and other great features'
+                />
+            </div>
+            <button className='UpgradeMattermostCloud__upgradeButton'>
+                <FormattedMessage
+                    id='admin.billing.subscription.upgradeMattermostCloud.upgradeButton'
+                    defaultMessage='Upgrade Mattermost Cloud'
+                />
+            </button>
+        </div>
+    );
+
+    const privateCloudCard = () => (
+        <div className='PrivateCloudCard'>
+            <div className='PrivateCloudCard__text'>
+                <div className='PrivateCloudCard__text-title'>
+                    <FormattedMessage
+                        id='admin.billing.subscription.privateCloudCard.title'
+                        defaultMessage='Looking for a high-trust private cloud?'
+                    />
+                </div>
+                <div className='PrivateCloudCard__text-description'>
+                    <FormattedMessage
+                        id='admin.billing.subscription.privateCloudCard.description'
+                        defaultMessage='If you need software with dedicated, single-tenant architecture, Mattermost Private Cloud (Beta) is the solution for high-trust collaboration.'
+                    />
+                </div>
+                <button className='PrivateCloudCard__contactSales'>
+                    <a
+                        href={contactSalesLink}
+                        rel='noopener noreferrer'
+                        target='_blank'
```

Comment:
all these `_blank` <a> elements would be worth checking if they work as expected on the desktop client.

Reply:
Good catch! Fixed them all to use `target='_new'` which does work properly on both webapp and desktop 👌